src/include/sof/panic.h

```diff
@@ -37,7 +37,7 @@ struct sof_ipc_panic_info;
 
 void dump_panicinfo(void *addr, struct sof_ipc_panic_info *panic_info);
 void panic_rewind(uint32_t p, uint32_t stack_rewind_frames,
-		  struct sof_ipc_panic_info *panic_info);
+		  struct sof_ipc_panic_info *panic_info, uintptr_t *epc1);
```

Comment:
epc1 needs to be renamed to data since this is a generic non xtensa header.

Reply:
Ok, done